src/resources.cpp

```diff
@@ -103,13 +107,13 @@ class FileResourceStream : public ResourceStream
 
 
 DirectoryResourceProvider::DirectoryResourceProvider(string basepath)
-: basepath(basepath)
 {
+  this->basepath = basepath.endswith("/") ? basepath.substr(0,basepath.size()-1) : basepath;
```

Comment:
wrong amount of indentation spaces (2 instead of 4)

You could use `rstrip("\\/")` instead, which is less code and removes both windows and linux path seperators.

Reply:
Ok, also used this nice helper for the EE part.